assembly/src/assembler/instruction/adv_ops.rs

```diff
@@ -21,16 +21,8 @@ pub fn adv_push(span: &mut SpanBuilder, n: u8) -> Result<Option<CodeBlock>, Asse
 // ADVICE INJECTORS
 // ================================================================================================
 
-/// Appends adv.mem.a.n advice injector to the span. This operation copies n number of words from
+/// Appends adv.mem advice injector to the span. This operation copies n number of words from
 /// memory the starting at address a into the advice provider's key-value map.
-///
-/// # Errors
-/// Returns an error is start_addr + num_words > u32::MAX.
```

Comment:
This requirement didn't make much sense to me, we currently don't limit the address to u32 [ref](https://github.com/0xPolygonMiden/miden-vm/issues/833#issuecomment-1502681256), so I removed it. (It can't be implemented here, but it could be done in the processor during runtime).